packages/trip-form/src/MetroModeSelector/AdvancedModeSubsettingsContainer.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import React, { useCallback } from "react";
+import { ModeButtonDefinition } from "@opentripplanner/types";
+import colors from "@opentripplanner/building-blocks";
+import AdvancedModeSettingsButton from "./AdvancedModeSettingsButton";
+import { invisibleCss } from ".";
+import { QueryParamChangeEvent } from "../types";
+
+const { grey } = colors;
+
+const SubsettingsContainer = styled.fieldset`
+  border: none;
+  margin: 0;
+
+  legend {
+    ${invisibleCss}
+  }
+
+  display: flex;
+  flex-direction: column;
+
+  div:first-of-type div label {
+    border-top-width: 2px;
+    border-radius: 8px 8px 0 0;
+  }
+
+  div:last-of-type div label {
+    border-bottom-width: 2px;
+    border-radius: 0 0 8px 8px;
+  }
+
+  div.advanced-submode-container:last-of-type div.subsettings-container {
```

Comment:
Is it possible to put this style in the styled component for the container instead?

Reply:
(is there a pseudo selector for that?)

Reply:
Unfortunately no, because the subsettings container is only aware of itself so it can't identify when it's the last of type. I think it has to happen here or higher!